QuotationsWebApi/Dtos/QuotationDtos/QuotationGetDto.cs

```diff
@@ -0,0 +1,19 @@
+﻿using QuotationsWebApi.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace QuotationsWebApi.Dtos
+{
+    public class QuotationGetDto
+    {
+        public Guid Id { get; set; }
```

Comment:
you can add some attributes to those fields to help with the generation of the database.
[Key]
[DatabaseGenerated(DatabaseGeneratedOption.Identity)]